packages/office-addin-dev-settings/src/cli.ts

```diff
@@ -26,7 +26,9 @@ commander
 commander
     .command("debugging <manifest-path>")
     .option("--enable", `Enable debugging for the add-in.`)
-    .option("--disable", "Disable debugging for the add-in")
+    .option("--disable", "Disable debugging for the add-in.")
+    .option("--ie", `Enable debugging for the add-in on IE.`)
+    .option("--edge", "Disable debugging for the add-in on IE.")
```

Comment:
This is just for marking add-ins to have debugging enabled. For the choice of the webview to use, this should be on the start command in office-addin-debugging.

Reply:
Are you going to revert this part?

Reply:
Removed in latest commit